package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "hackmud_wiki",
+  "version": "0.0.1",
+  "description": "Knowledge repository for the hackmud video game",
+  "author": "ComCODE",
+  "license": "CC-BY-NC-SA-4.0",
+  "private": true,
+  "scripts": {
+    "start": "docusaurus start",
+    "build": "docusaurus build",
+    "swizzle": "docusaurus swizzle",
+    "clear": "docusaurus clear",
+    "serve": "docusaurus serve",
+    "write-translations": "docusaurus write-translations",
+    "write-heading-ids": "docusaurus write-heading-ids",
+    "lint": "prettier --check .",
+    "fix": "prettier --write ."
+  },
+  "dependencies": {
+    "@docusaurus/core": "^2.4.3",
+    "@docusaurus/plugin-content-docs": "^2.4.3",
+    "@docusaurus/theme-classic": "^2.4.3"
+  },
+  "devDependencies": {
+    "@docusaurus/module-type-aliases": "^2.4.3",
+    "@tsconfig/docusaurus": "^2.0.1",
+    "prettier": "^3.0.3",
+    "typescript": "^5.2.2"
+  },
+  "engines": {
+    "node": ">=16.14"
```

Comment:
Any reason we're allowing node 16 here?

Likely we should discuss node version management (nvm) -- though not sure how this stuff is looking on windows these days and if it's gotten better--

Reply:
This is just the minimum required version of Node for the current version of Docusaurus. I didn't mean to imply that 16.x should actually be used for development.

I also have little experience with Node on Windows. I've used [`fnm`](https://github.com/Schniz/fnm) though, which alleges to be cross-platform.

Reply:
Trade-off we need to consider is: having a lower node req allows us to support more outside contributors, but would add to our testing matrix for github actions.

Reply:
Ah yes, 16 being EOL is another good reason. I'll get on board. Bumping up to 20.x is probably right, if everything builds correctly and there's not a dependency that blows up when we change this requirement. There's a bit to be said about 20.x not having a lot of bake time at this point, but probably ok.

Reply:
To be clear: let's move this line to 20.8 (current latest 20.x), then I think we are good to go.

Reply:
Bumped to `>=20.x` in https://github.com/comcode-org/hackmud_wiki/pull/2/commits/527ceec7bc6c11d23e77703067e0ecdf75204607.
I left it as >=, which may soon require a test matrix for V21 if we want to support non-LTS versions, or we can change the comparator if you want.

Reply:
Nope leave the comparator-- we get to choose which versions of node we run against in the actions workflow. This gives us what we want.